easy-ui-react/src/IconButton/IconButton.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import AddIcon from "@easypost/easy-ui-icons/Add";
```

Comment:
for some reason im unable to view this story via the preview link, do you know why that might be?

![Screen Shot 2023-04-11 at 8 41 08 AM](https://user-images.githubusercontent.com/44451175/231216078-4aeae63a-3908-497c-a84a-03017e7bb29d.png)

Reply:
nice catch. had a typo in the story file. even with the experimental MDX language server enabled in VSCode, i'm still finding it lacking with catching errors like that :(